templates/main.php

```diff
@@ -167,46 +167,46 @@ class="collection"
             <div class="app-navigation-entry-menu bubble sortorder-dropdown">
```

Comment:
This is NOT a `app-navigation-entry-menu`, we're in the content, this should be a popovermenu 😉 
Though this should roughly be similar, changing the class should be enough :)

The menu content on the other hand, seems to be broken (triangle), so since we're working on this, maybe fix this too? 😄